leap.js

```diff
@@ -19,7 +19,7 @@ function Year(year) {
 Year.prototype.isLeap = function () {
 
 
-  function leapYear(year)
+  function this.year
```

Comment:
Nope :disappointed: .. don't change the function name, change function variable calls inside !